src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -166,6 +167,15 @@ export default class Mutations extends React.Component<
         updateOncoKbIconStyle({ mergeIcons });
     }
 
+    @autobind
+    protected isPutativeDriver(m: AnnotatedMutation) {
+        return (
+            m.putativeDriver ||
+            (getProteinImpactType(m.mutationType) === 'other' &&
+                !!m.oncoKbOncogenic)
+        );
+    }
```

Comment:
This seems like duplicate of https://github.com/cBioPortal/cbioportal-frontend/pull/4524/files#diff-da875ea197b72753578a956f5402607a962e932caefd8f73d2030ca70af24b07R120-R127. Better to move into a utility function and reuse.